includes/core/server/class-request.php

```diff
@@ -408,7 +438,7 @@ final protected function unset_param( string $key ) {
 	 *
 	 * @param array $params The new parameters to use.
 	 */
-	private function set_params( $params ) {
+	public function set_params( $params ) {
```

Comment:
Same commend about making this one public.

Reply:
Addressed here: https://github.com/Automattic/woocommerce-payments/pull/7314/commits/6a78ee9776f280116ebe4519e649cb2d692f0eeb